index.html

```diff
@@ -228,5 +228,37 @@ <h3>Languages</h3>
             </div>
         </div>
     </div>
-</body>
-</html>
\ No newline at end of file
+    <form class="former-block">
+                <div class="form-block">
+                    <section>
+                        <h1 class="contact">Contact me</h1>
+                            <p class="paragraph">If you have an application y ou are interested in developing,
+                                a feature that you need built or   a project that needs 
+                                coding. I’d love to help with it 
+                            </p>
```

Comment:
- [ ] Kindly add some spacing around the text (hint: `padding`) to match the design in Figma 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/200881916-33984a23-dc5f-4748-8aae-b5f1a7d4df53.png)

## Figma design
![image](https://user-images.githubusercontent.com/50721479/200881957-3b8b8d20-aa5b-4c29-b4f4-05feedb7fd9d.png)